domains/nevo.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NevoM4",
+        "email": "nevomordechai@gmail.com"
+    },
+    "record": {
```

Comment:
Invalid URL record (missinf protocol)

also you can't just point your domain to google